supportconfig/plugins/suse_public_cloud

```diff
@@ -301,6 +301,18 @@ function update_infra_data() {
     zypper ref >> $INFO_FILE
 }
 
+function csp_billing_adapter() {
+    local INFO_FILE="$DATA_LOCATION/csp_billing_adapter_config.txt"
+    local BA_CONFIG_FILE="/etc/csp_billing_adapter/config.yaml"
+    echo "#==[ CSP billing adapter config ]========================#" > $INFO_FILE
+    if [ -e $BA_CONFIG_FILE ]; then
+        echo "# cat $BA_CONFIG_FILE" >> $INFO_FILE
+        cat $BA_CONFIG_FILE >> $INFO_FILE
+    else
+        echo "# not found" >> $INFO_FILE
+    fi
```

Comment:
We also want the log file the billing adapter creates.

Reply:
What file would that be?

Reply:
Where ever @jesusbv who is implementing the -local csp adapter part decides to write the logging information. Somewhere in `/var/log`. `cspadapate.log` would make sense to me but I have not checked up on the state of the `-local` implementation for the adapter.

Reply:
File based logging is not implemented yet. Based on other naming choices I would suggest something along the lines of `/var/log/csp_billing_adapter.log` or `/var/log/csp-billing-adapter.log`

Reply:
@jgleissner pick one either '_' or '-' then it is prescribed for the local logging implementation, whomever does it has to use the name picked here.